english/batman/batman.md

```diff
@@ -10,9 +10,7 @@ Click [here](http://www.imdb.com/title/tt0096895/) to visit IMDB.
 
 But I know what you're wondering. If Bruch Wayne is the Bat, then who is Bruce Wayne? Well it's obvious...
 
-...IT'S JOHN CENA!!!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.